gems/manageiq-providers-amazon/app/models/manageiq/providers/amazon/cloud_manager.rb

```diff
@@ -20,6 +20,9 @@ class ManageIQ::Providers::Amazon::CloudManager < ManageIQ::Providers::CloudMana
   require_nested :Template
   require_nested :Vm
 
+  # register as a manager_type
+  OrchestrationTemplateCfn.eligible_manager_types << self
```

Comment:
@durandom somehow this feels wrong.  I would have preferred something like:
```ruby
OrchestrationTemplateCfn.register_eligible_manager_type(self)
```
/cc @Fryguy @matthewd